main_logic/core.py

```
@@ -945,16 +945,25 @@ def _start_tts_thread(self):
-            core_api_type=self.core_api_type,
-            has_custom_voice=has_custom,
-            voice_id=self.voice_id or '',
-        )
-        tts_config = self._config_manager.get_model_api_config(
-            'tts_custom' if has_custom else 'tts_default'
-        )
-        api_key = api_key_override or tts_config['api_key']
+        # 检查是否禁用了 TTS
+        core_config = self._config_manager.get_core_config()
+        if core_config.get('DISABLE_TTS', False):
+            logger.info("TTS 已被用户禁用，使用 dummy worker")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)